M2/Macaulay2/m2/overview.m2

```diff
@@ -1,1160 +0,0 @@
-     }
-
-document { "making random matrices",
-     "The ", TO "random", " command can be used in various ways to make random
-     matrices.  We could assemble random polynomials into a doubly nested list
-     and use ", TO "matrix", " to enter them into a matrix.",
-     EXAMPLE {
-	  "R = ZZ/101[x,y,z];",
-      	  "matrix table(3,3,(i,j)->random(2,R))",
-      	  "matrix table(3,3,(i,j)->random(i-j,R))",
-	  },
-     NOINDENT, "As usual, the degrees of the basis elements of the source and target
-     of the map are determined from the polynomials' degrees.  Alternatively,
-     one could generate a free module with the appropriate degrees and use
-     ", TO "random", " to insert random matrix entries of the appropriate
-     degree so as to make the matrix homogeneous.",
-     EXAMPLE {
-	  "random(R^3,R^{1,0,-1,-2})",
-      	  "random(R^{1,0,-1},R^{1,0,-1})"
-	  },
-     }
-
-document { "making generic matrices",
-     "Here a few commands for making various sorts of generic matrices.  A generic
-     matrix is one whose entries are independent variables from the ring, subject
-     to certain relations.",
-     PARA,
-     "We begin by making a ring with enough variables to accommodate all
-     the examples.",
-     EXAMPLE "R = ZZ/101[a..z];",
-     "We can make a general generic matrix with ", TO "genericMatrix", ".  We specify
-     the ring, the starting variable and the dimensions.",
-     EXAMPLE "genericMatrix(R,c,3,5)",
-     "We can also make a skew symmetric matrix with ", TO "genericSkewMatrix", " or
-     a symmetric matrix with ", TO "genericSymmetricMatrix", ".",
-     EXAMPLE {
-	  "R = ZZ/101[a..i];",
-      	  "genericSkewMatrix(R,c,3)",
-      	  "gs = genericSymmetricMatrix(R,a,3)",
-	  },
-     "Suppose we need a random symmetric matrix of linear forms in three variables.
-     We can use ", TO "random", " and ", TO "substitute", " to obtain it from the
-     generic symmetric matrix ", TT "gs", " above.",
-     EXAMPLE {
-	  "S = ZZ/101[x,y,z];",
-      	  "rn = random(S^1, S^{9:-1})",
-      	  "substitute(gs, rn)"
-	  },
-     }
-
-document { "displaying matrices",
-     "Normally matrices are displayed in compact notation that originated 
-     with Macaulay.",
-     EXAMPLE {
-	  "R = ZZ/101[x,y];",
-	  "f = random(R^1,R^{2:-2})",
-	  },
-     "Setting the global flag ", TO "compactMatrixForm", " to ", TO "false", "
-     will modify that behavior so that matrices are displayed more clearly
-     and less compactly.",
-     EXAMPLE {
-	  "compactMatrixForm = false",
-	  "f",
-	  },
-     "We may use ", TO "toString", " to produce a string, useful as input into
-     other programs, or into Macaulay 2 at another time.",
-     EXAMPLE "toString f",
-     "The function ", TO "toExternalString", " tries harder: it gives
-     complete information about the source, target, and degree of the map.",
-     EXAMPLE "toExternalString f"
-     }
-
-document { "determinants",
```

Comment:
Did you check that the information in this document is preserved elsewhere? As you pointed out, I listed this file as stale in 2020, at which point I could have also removed them myself. The reason I didn't was that I read this file and learned things from it that as far as I know are still not in the main documentation. This node in particular comes to mind, documenting determinants and their options.